cmd/main.go

```diff
@@ -31,19 +32,30 @@ func main() {
 
 	r := chi.NewRouter()
 
-	db, err := sql.Open("postgres", cfg.DBConnString)
+	client, err := mongo.NewClient(options.Client().ApplyURI(cfg.DBConnString))
 	if err != nil {
 		logger.Fatal().Err(err).Msg("DB initializing error")
 	}
-	defer db.Close()
 
-	err = db.Ping()
+	err = client.Connect(context.TODO())
```

Comment:
Not sure why are you using context.TODO(), isn`t it better to create an implementetion of context with timeout and provide it into the functions

Reply:
You're right. Thanks